### DISCUSSION TITLE
Add support for podcasts in quarto blog

### DISCUSSION THREAD

**Discussion by produnis** [quarto-dev/quarto-cli#8474] [Feature Requests]
### Description

Hi there,
I'm wondering if it's possible to incorporate podcast support for Quarto blogs.

It's worth noting that many podcast producers utilize WordPress with plugins, as a podcast essentially boils down to being an `RSS` feed with specific fields.

In my understanding, it seems that adding podcast functionality could be achieved by including an entry like the following within the `<item>` tag of the RSS feed:

`<enclosure length="SECONDS" type="audio/mpeg" url="MY-FILE.mp3"/>`

To ensure compatibility with platforms such as iTunes, additional tags like `<itunes:title>`, `<itunes:author>` and `<itunes:summary>` must be populated.

I believe integrating podcast support into Quarto blogs would be a valuable addition.

**Comment by bryanwhiting** [comment]
As of today, you can embed audio https://www.bryanwhiting.com/creative/guitar-clouds.html. 

Tutorial here showing 3 ways: https://www.bryanwhiting.com/tech/embedding-audio-in-quarto.html

**Reply by produnis** [reply]
Thx for your answer. My Suggestion is about putting the Audio link inside a special field of the RSS-feed, so that you could add it to your favorite pod fetcher.

**Reply by RichardJActon** [reply]
Nice. This would make it easier to have a player on a corresponding episode notes page of a podcast RSS feed but not solve the issue of actually generating that feed. I've also suggested this here:https://github.com/quarto-dev/quarto-cli/discussions/3623

**Reply by durraniu** [reply]
This site just shows me ads.

**Reply by RichardJActon** [reply]
> This site just shows me ads.

which site?

**Reply by durraniu** [reply]
Bryan whiting.com

**Comment by mcanouil** [comment]
Thanks, I marked as spam as it seems the domain is no longer own by the original poster.
